<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogPacketParser</class>
 <widget class="QDialog" name="DialogPacketParser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Packet parser (test)</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPacketHexView">
     <item>
      <widget class="QLabel" name="labelDesc_lineEditPacketHexView">
       <property name="text">
        <string>hex</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPacketHexView">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeViewPacketFields"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutOK">
     <item>
      <widget class="QLineEdit" name="lineEditParsingResult">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonOK">
       <property name="styleSheet">
        <string notr="true">QPushButton {
	min-size = 30px;
}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
